README.md

```diff
@@ -151,6 +151,23 @@ $ brew install curl (Homebrew)
 $ port install curl (MacPorts)
 ```
 
+#### Arch Linux / Manjarao (Arch based)
```

Comment:
Should it be `Manjaro`?

Reply:
> We'll just assume most users can respond to the password prompt, but, it might be better to split the package making and package installation into two things.

Now 

> We'll just assume most users can respond to the password prompt, but, it might be better to split the package making and package installation into two things.

now all should be fine?